codegen/sync.act

```diff
@@ -592,9 +592,6 @@ action untrustThisKey
         return status;
 
     OCTET_STRING_fromBuf(&session->sync_state.keysync.key, "", 0);
-
-    exit(0);
```

Comment:
This was added on [CORE-65](https://github.com/plancksecurity/foundation-planckCoreV3/commit/dd40a0033abb21e1a0ccd5776d9822f75e24bed9) why exactly is it unneeded now?

Reply:
It's very clearly "debugging" code that should never have reached production. You don't hard `exit` in a production build of a library.

Reply:
Agreed, I was just curious since you added it so maybe there was a deeper reasoning.